### ISSUE TITLE
Missing `imageUri`

### ISSUE THREAD

**Issue by mgsgde** [francescov1/facebook-event-scraper#14]
#### Description:

In the responste photo object, the `imageUri` is returned as `undefined` even though the corresponding `img` element can be retrieved using the CSS selector `[data-imgperflogname="profileCoverPhoto"]`.

```javascript
photo: {
    url: 'https://www.facebook.com/photo/?fbid=770779404842505&set=gm.835134241383644',
    id: '770779404842505',
    imageUri: undefined
}
```

#### Expected Behavior:
The `imageUri` should correctly reflect the `src` attribute of the retrieved `img` element.

#### Steps to Reproduce:
```
import { expect, test } from '@playwright/test';
import { scrapeFbEvent } from 'facebook-event-scraper';

const EVENT_URL = 'https://www.facebook.com/events/835134204716981';

test('fetch event details and image', async ({ page }) => {
    const eventData = await scrapeFbEvent(EVENT_URL);
    console.log('Event Data:', eventData);
    
    const imageUrl = await extractImageFromPage(page);
    console.log('Image URL:', imageUrl);

    expect(eventData.photo.imageUri === imageUrl).toBe(true)
});

async function extractImageFromPage(page) {
    await page.goto(EVENT_URL);
    await page.content();
    const element = await page.$('[data-imgperflogname="profileCoverPhoto"]');
    return await element.getAttribute('src');
}
```
#### Proposed Solutions:
- Assign the `src` attribute of the `<img>` element, which can be selected using the CSS selector `[data-imgperflogname="profileCoverPhoto"]`, to the `imageUri` field.

**Comment by francescov1** [comment]
Hey @mgsgde, nice catch! Turns out that the image URI can sometimes be set under a field called `full_image` rather than the standard `image` field in the FB JSON data [here](https://github.com/francescov1/facebook-event-scraper/blob/be2d73ae918fa0bd4a2d3c8c00ec69848e35d309/src/utils/htmlParser.ts#L52C4-L52C4). I'll put up a PR shortly!

**Comment by francescov1** [comment]
https://github.com/francescov1/facebook-event-scraper/pull/15/files

**Comment by francescov1** [comment]
Fix is live in v0.0.13
